### PR TITLE
Update to Presentation Exchange v2 and Latest CM

### PR DESCRIPTION
**Pull request by decentralgabe**

### CODE REVIEW

File: credential/manifest/builder_test.go [hunk 1]
```diff
@@ -18,6 +18,12 @@ func TestCredentialManifestBuilder(t *testing.T) {
 
 	assert.False(t, builder.IsEmpty())
 
+	err = builder.SetName("name")
+	assert.NoError(t, err)
+
+	err = builder.SetDescription("description")
+	assert.NoError(t, err)
```
Comment (andresuribe87) [inline credential/manifest/builder_test.go:25]: Should this test verify that the correct values are set when the CM is built?

**Review by nitro-neal — APPROVED** [review]
:lit:

File: schema/known_schemas/pe-presentation-submission.json [hunk 2]
```diff
@@ -28,7 +28,7 @@
           "$ref": "#/definitions/descriptor"
         },
         "format": {
-          "$ref": "https://identity.foundation/claim-format-registry/schemas/presentation-submission-claim-format-designations.json#/definitions/format"
+          "$ref": "http://identity.foundation/claim-format-registry/schemas/presentation-submission-claim-format-designations.json#/definitions/format"
```
[the remark this answers is not in the document]
Comment (decentralgabe) [reply schema/known_schemas/pe-presentation-submission.json:31]: json schema breaks when you do 😢

File: credential/manifest/builder_test.go (hunk 1 above)
Comment (decentralgabe) [reply credential/manifest/builder_test.go:25]: added

**Pull request merged**